ludwig/utils/backward_compatibility.py

```diff
@@ -848,6 +849,21 @@ def remove_extra_type_param_in_defaults_config(defaults: FeatureTypeDefaultsDict
     return defaults_copy
 
 
+@register_config_transformation("0.8")
```

Comment:
@arnavgarg1 @tgaddair not actually sure if this config upgrade is necessary, can I delete it?

also note to self: need to update this as well.

Reply:
I think we can get rid of it since we haven't pushed out Ludwig 0.8 yet!

Reply:
removed